mix_helpers.exs

```diff
@@ -747,3 +751,7 @@ defmodule Mix.Appsignal.Helper do
   defp serialize_report_value(value) when is_binary(value), do: value
   defp serialize_report_value(value), do: inspect(value)
 end
+
+with [{Appsignal.Utils = module, _}] <- required_utils do
+  :code.delete(module)
```

Comment:
This would be the first time we load and unload modules in the mix_helpers. We tried it before and ran into some trouble with apps not reloading properly. Did you run into any issues or warnings with one of the test apps?

Reply:
Patch looks good, but I’m also hesitant about unloading that module. Another option would be to duplicate the function we need into the mix_helpers.exs file, as we’ve done before to work around this.

Reply:
@jeffkreeftmeijer I'm not happy with it either, and @tombruijn was spot on: there are warnings when running it on the test apps. I've gone ahead and duplicated it into the `mix_helpers.exs` file.